CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+3.0.0 Release notes (YYYY-MM-DD)
+=============================================================
+### Compatibility
+* Sync protocol: 24
+* Server-side history format: 4
+* File format: 7
+* Realm Object Server: 3.0.0 or later
+
+The feature known as Partial synchronization has been renamed to Query-based synchronization and is now the default mode
+for synchronized Realms. This has impacted a number of APIs. See below for the details.
+
+### Breaking changes
+
+* [Sync] `Realm.Configuration.SyncConfig._disablePartialSyncUrlChecks` has been renamed to `Realm.Configuration.sync._disableQueryBasedSyncUrlChecks`.
```

Comment:
This is an internal/private setting so I will not consider it breaking. I think it can be move to the *Internal* section.